buddyboss/buddyboss-platform
# 1299 - Fixed the notification date

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1299 

### How to test the changes in this Pull Request:

1. Inside admin, Settings > General, set the timezone to UTC -4 or Los Angeles, or whatever timezone except UTC+0
2. Post a reply using a different account
3. Login to the account that started the discussion and you should see a notification
4. Click the notification bell located at the top right corner of the page and you'll see that the timestamp is incorrect.

### Proof Screenshots or Video
https://www.loom.com/share/cc27a971ee6243c49cedcfdb4be68b19



### Other information:
